R/RLearner_classif_glmboost.R

```diff
@@ -4,27 +4,37 @@ makeRLearner.classif.glmboost = function() {
     cl = "classif.glmboost",
     package = "mboost",
     par.set = makeParamSet(
-      makeDiscreteLearnerParam(id = "family", default = mboost::Binomial(), values = list(AdaExp = mboost::AdaExp(), Binomial = mboost::Binomial())),
+      makeDiscreteLearnerParam(id = "family", default = "Binomial",
+        values = c("Binomial", "AdaExp", "AUC", "PropOdds", "custom.family")),
       # FIXME default of glmboost() for family is Gaussian()
+      makeUntypedLearnerParam(id = "custom.family.definition", requires = quote(family == "custom.family")),
+      makeNumericVectorLearnerParam(id = "nuirange", default = c(-0.5,-1), requires = quote(family == "PropOdds")),
+      makeNumericVectorLearnerParam(id = "offrange", default = c(-5,5), requires = quote(family == "PropOdds")),
+      makeDiscreteLearnerParam(id = "Binomial.link", default = "logit", values = c("logit", "probit")),
       makeIntegerLearnerParam(id = "mstop", default = 100L, lower = 1L),
       makeNumericLearnerParam(id = "nu", default = 0.1, lower = 0, upper = 1),
       makeDiscreteLearnerParam(id = "risk", values = c("inbag", "oobag", "none")),
       makeLogicalLearnerParam(id = "stopintern", default = FALSE),
```

Comment:
pls just keep the params for completeness sake. you never know what the user wants to do with the fitted model aftwerwards.